doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -3762,27 +3760,27 @@ The execution of the Ltac expression:
 
 :n:`under @term => [ @i_item__1 | … | @i_item__n ].`
 
-involves the following steps:
+involves the following steps.
 
 1. It performs a :n:`rewrite @term`
    without failing like in the first example with ``rewrite eq_map.``,
    but creating evars (see :tacn:`evar`). If :n:`term` is prefixed by
    a pattern or an occurrence selector, then the modifiers are honoured.
 
-2. As a n-branches intro pattern is provided :tacn:`under` checks that
+2. As a n-branch intro pattern is provided, :tacn:`under` checks that
```

Comment:
```suggestion
2. As an n-branch intro pattern is provided, :tacn:`under` checks that
```

"an" because the pronunciation of "n-branch" begins with a vowel ("en-branch").